docs/en/events.rst

```diff
@@ -30,45 +30,73 @@ Add a new listener with the event manager:
 
 .. code-block:: php
 
-    use App\Listeners\PostParseDocumentListener;
+    use App\Listeners\DocumentListener;
     use Doctrine\RST\Event\PostParseDocumentEvent;
 
     $eventManager->addEventListener(
-        [PostParseDocumentEvent::POST_PARSE_DOCUMENT],
-        new PostParseDocumentListener()
+        [PostParseDocumentEvent::POST_PARSE_DOCUMENT /*, other events, if any*/],
+        new DocumentListener()
     );
 
-Now define your listener in ``App\Listeners\PostParseDocumentListener``. The ``postParseDocument()``
-method will be notified every time a document is parsed:
+Now, define your listener in ``App\Listeners\DocumentListener``. The ``postParseDocument()``
+method will be invoked every time a document is parsed:
 
 .. code-block:: php
 
     namespace App\Listeners;
 
-    use Doctrine\RST\Event\PostParseDocumentEvent;
     use Doctrine\RST\Event\PostParseDocumentEvent;
 
-    class PostParseDocumentListener
+    class DocumentListener
     {
         public function postParseDocument(PostParseDocumentEvent $event)
         {
             $documentNode = $event->getDocumentNode();
 
             // do something with $documentNode
         }
+        
+        // Other event handlers, if any
+        // ...
     }
 
 Available Events
 ----------------
 
-The events you can listen for are as follows:
-
-- ``PreBuildScanEvent::PRE_BUILD_SCAN`` - Dispatches a method named ``preBuildScan()`` before files are scanned when using the builder.
-- ``PreBuildParseEvent::PRE_BUILD_PARSE`` - Dispatches a method named ``preBuildParse()`` before files are parsed and after they are scanned when using the builder.
-- ``PreBuildRenderEvent::PRE_BUILD_RENDER`` - Dispatches a method named ``preBuildRender()`` before files are rendered and after they are parsed when using the builder.
-- ``PostBuildRenderEvent::POST_BUILD_RENDER`` - Dispatches a method named ``postBuildRender()`` after files are rendered when using the builder.
-- ``PostNodeCreateEvent::POST_NODE_CREATE`` - Dispatches a method named ``postNodeCreate()`` after a node is created.
-- ``PreParseDocumentEvent::PRE_PARSE_DOCUMENT`` - Dispatches a method named ``preParseDocument()`` before a node is parsed.
-- ``PostParseDocumentEvent::POST_PARSE_DOCUMENT`` - Dispatches a method named ``postParseDocument()`` after a node is parsed.
-- ``PreNodeRenderEvent::PRE_NODE_RENDER`` - Dispatches a method named ``preNodeRender()`` before a node is rendered.
-- ``PostNodeRenderEvent::POST_NODE_RENDER`` - Dispatches a method named ``postNodeRender()`` after a node is rendered.
+The events you can listen for and their respective handlers:
+
+.. list-table:: 
+   :widths: 2 1 4
+   :header-rows: 1
+   :stub-columns: 1
+
```

Comment:
I don't think this directive is yet supported by the RstParser - at least a simple `grep list-table` didn't give any results.

It would be a nice directive to support though :)

Reply:
I can replace this ``list-table`` with pseudo-graphics table made out of `+`, `-`, `|` and `=` characters. Shall I do it?

Reply:
ping @wouterj

Reply:
In the T3Docs we are offering 4 different table formats: 

https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Reference/Content/Tables.html

For the complicated format there is a table editor: https://www.tablesgenerator.com/text_tables

Reply:
However with the data in Question I would probably not use a table but a format like confval (https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Reference/Code/Confval.html)